loadrunner_professional/assets/logs/loadrunner_professional_tests.yaml

```diff
@@ -0,0 +1,77 @@
+id: "loadrunner-professional"
```

Comment:
Same as before the file should be renamed with the `<integrationid>_tests.yaml`.

That should trigger the execution of the tests.